oneview_redfish_toolkit/blueprints/network_interface.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import logging
+
+# 3rd party libs
+from flask import abort
+from flask import Blueprint
+from flask import Response
+from flask_api import status
+
+# own libs
+from hpOneView.exceptions import HPOneViewException
+from oneview_redfish_toolkit.api.network_interface import NetworkInterface
+from oneview_redfish_toolkit import util
+
+
+network_interface = Blueprint("network_interface", __name__)
+
+
+@network_interface.route(
+    "/redfish/v1/Systems/<uuid>/NetworkInterfaces/<device_id>",
```

Comment:
Missing **/** at the end of the route.

Reply:
You can change .../<device_id> to .../<int:device_id>, so the cast is no longer needed.

Reply:
If I do that I need to cast it back to string in other places. Going to keep it a string.
Will do validation here not in the class.